packages/attestation-service/src/attestation.ts

```diff
@@ -0,0 +1,39 @@
+import { attestToIdentifier, SignatureUtils } from '@celo/utils'
+import express from 'express'
+import { sendSms } from './sms'
+
+function signAttestation(phoneNumber: string, account: string) {
+  if (process.env.ATTESTATION_KEY === undefined) {
+    console.error('Did not specify ATTESTATION_KEY')
+    throw new Error('Did not specify ATTESTATION_KEY')
+  }
+
+  const signature = attestToIdentifier(phoneNumber, account, process.env.ATTESTATION_KEY)
+
+  return SignatureUtils.serializeSignature(signature)
+}
+
+function toBase64(str: string) {
+  return Buffer.from(str, 'hex').toString('base64')
+}
+
+function createAttestationTextMessage(attestationCode: string) {
+  return `<#> ${toBase64(attestationCode)} ${process.env.APP_SIGNATURE}`
+}
+
+export async function handleAttestationRequest(req: express.Request, res: express.Response) {
+  // TODO: Should parse request appropriately
```

Comment:
Suggest adding error handling at this level

Reply:
Agreed, and have a separate ticket for it https://github.com/celo-org/celo-monorepo/issues/1075

Reply:
It's not just about malformed requests, if you code throws an exception somewhere (like when sending with Nexmo), that will propagate all the way up and the 500 we return will leak details about the internal error

Reply:
My goal would it be to handle this more globally with that ticket. You are right that the ticket did not specify that, I edited the ticket